lms/djangoapps/django_comment_client/utils.py

```diff
@@ -137,6 +138,29 @@ def get_accessible_discussion_modules(course, user, include_all=False):  # pylin
     ]
 
 
+def get_discussion_module(course, user, discussion_id):
```

Comment:
@robrap I was unaware of existence of `discussion_category_id_access`; and using this method instead of has_access would be more efficient as it uses cache.
I would need to refactor this method a little to make use of it. Will update.